Assignment_1.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+import sys
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import Normalizer
+
+
+def print_heading(title):
```

Comment:
This is good, this way by calling the 'print_heading' function you can easily differentiate between different sections.